src/request-state.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+
+const useIsFetching = () => useState(false);
+const useIsFetched = () => useState(false);
+const useData = () => useState(null);
+const useError = () => useState(null);
+const useTimerSignal = () => useState(0);
```

Comment:
I'm all for making things explicit, but these alias functions seem like a bit much to me.

Reply:
This makes them show up with labels in React dev tools.

before:
```
State: false
State: null
State: null
State: 0
```

after:
```
IsFetched: false
Data: null
Error: null
TimerSignal: 0
```

Reply:
seems like a problem with devtools to me....

but whatevs 🤷‍♂️